fern/pages/accounts/deposits-and-withdrawals/email-wallets.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Email Wallets
+hide-feedback: true
```

Comment:
we can pull this out now. we used to have this before we setup the feedback items, but now it's working.